### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 3 - Esercizio 5

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#504] [Esercizi M-Z]
![es.5](https://github.com/user-attachments/assets/2ec3aed4-48c0-4302-bc18-36df7abab54a)

**Comment by AglaiaNorza** [comment]
dimostrazione svolta in classe dal professor Pellarin:

![PTF](https://github.com/user-attachments/assets/f6f65632-8760-4163-8cb8-79d851374bde)

**Comment by marcoseverini** [comment]
orribile
<img width="986" alt="immagine" src="https://github.com/user-attachments/assets/71ffd744-b010-4ddd-ad26-808d6e42a651">
<img width="696" alt="immagine" src="https://github.com/user-attachments/assets/bff232d6-7057-4300-adc8-d001a54921b9">
<img width="678" alt="immagine" src="https://github.com/user-attachments/assets/d211b74e-26d1-4f7d-a881-fe632d1cabfc">

**Reply by AglaiaNorza** [reply]
noo era molto fico !
